SECURITY.md

```diff
@@ -76,4 +105,31 @@ Extensions configuration setup is tied to OpenSearch settings, extensions config
 
 ## Reliability
 
-OpenSearch plugins can create cluster or node instability if incorrectly configured or by software defects.
\ No newline at end of file
+OpenSearch plugins can create cluster or node instability if incorrectly configured or by software defects.
+
+# Projects
+To stretch out the design process while fulfilling scenarios some security efforts will be tracked as longer running projects.  There will be tracking issues in github tied to the work, documentation here is justification for a project and how it ties into the security space.  Tasks will be denotated as incomplete with :negative_squared_cross_mark: `:negative_squared_cross_mark:` or completed with :white_check_mark:
+ `:white_check_mark:`.
+
+## Anomaly Detection Plugin
+Overall project is tracked with [[FEATURE] Migrate Anomaly Detector plugin to work as an Extension](https://github.com/opensearch-project/opensearch-sdk/issues/24).  By migrating this plugin it will exercise the general extensions and security specific scenarios.
```

Comment:
We can mention the meta issue for AD plugin to run as an extension. The first part of which is to start with the create detector functionality : https://github.com/opensearch-project/OpenSearch/issues/3635

Reply:
Thanks, I've updated the link to the more appropriate issue